<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SH-meiyanghua" name="test1" tests="3" failures="1" timestamp="31 Aug 2018 10:47:40 GMT" time="0.027" errors="0">
  <testcase name="f" time="0.008" classname="test.TestDemo1">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.0" classname="test.TestDemo1">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.001" classname="test.TestDemo1">
    <failure type="java.lang.AssertionError" message="expected [1] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [0]
	at test.TestDemo1.f(TestDemo1.java:11)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- test1 -->
